index.html

```diff
@@ -13,156 +22,139 @@
+                <li class="rubyraillist">Ruby on rails</li>
+                <li class="rspeclist">RSpec</li>
+                <li class="caplist">Capibara</li>
+                <li class="sellist">Selenium</li>
+            </ul>
             <div class="Skills_test">Skills</div>
             <img class="img9" src="Rectangle_42.png" alt="blue rectangle at bottom of page">
             <img class="img10" src="shape3.png" alt="Hash symbol">
-            <div class="table2">
-                <ul id="list-horizontal2">
-                    <li class="codekitlist">Codekit</li>
-                    <li class="codepenlist">Codepen</li>
-                    <li class="githublist">GitHub</li>
-                    <li class="gitlablist">GitLAb</li>
-                    <li class="termlist">Terminal</li>
-                </ul>
-            </div>
-        </article>
-
-        <div class="align-left-item">
-            <img class="img11" src="left_topside_icon.svg" alt="Image geometry on the left top side">
-        </div>
-        <div class="text10">Get started</div>
-        <div class="text11">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</div>
-        <section class="rectangle15">
-            <div class="text12">Your name</div>
-            <div class="text13">Anastasiia</div>
-        </section>
-        <section class="rectangle16">
-            <div class="text14">Your email</div>
+            <ul id="navigat2" class="list_items_languages">
+                <li class="codekitlist">Codekit</li>
+                <li class="codepenlist">Codepen</li>
+                <li class="githublist">GitHub</li>
+                <li class="gitlablist">GitLAb</li>
+                <li class="termlist">Terminal</li>
+            </ul>
         </section>
-        <section class="rectangle17">
-            <div class="text15">Enter your message here...</div>
-        </section>
-        <!--section class="rectangle18"!-->
-        <a href="#" class="border-button_startcolab">Start collaboration</a>
-            <!--div class="text16">Start collaboration</div!-->
-        <!--/section!-->
-        <section class="rectangle19">
+        <section class="Contact-form">
+            <img class="img11" src="left_topside_icon.svg" alt="Image geometry on the left top side">
+            <form name="Contactform" action="https://formspree.io/f/xbjqawyz" onsubmit="return validateForm()" method="post" aria-required>
+                <h1 class="text10">Get started</h1>
+                <p class="contact-form-para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                <ul class="contact-form-ullist">
+                    <li class="contact-form-name">
+                        <input type="text" maxlength="30" class="nooutline" id="name" name="user_name" placeholder="Your name">
```

Comment:
- [ ] The inputs inside the form should have the attribute `required`. This helps to validate a value on input with HTML5 help. It's easier than using an alert. Just add `required` inside the input tag. 🐱

Reply:
I have put the required in the input tag.